e2e/scenarios/kops-simple

```diff
@@ -127,43 +127,22 @@ if [[ -z "${IMAGE_TAG:-}" ]]; then
 fi
 echo "IMAGE_TAG=${IMAGE_TAG}"
 
-
 # Build and push cloud-controller-manager
 cd ${REPO_ROOT}
-export KO_DOCKER_REPO="${IMAGE_REPO}"
 
 export KUBE_ROOT=${REPO_ROOT}
 source "${REPO_ROOT}/tools/version.sh"
 get_version_vars
 unset KUBE_ROOT
 
-BUILD_DATE=$(date ${SOURCE_DATE_EPOCH:+"--date=@${SOURCE_DATE_EPOCH}"} -u +'%Y-%m-%dT%H:%M:%SZ')
-
-cat > ${WORKDIR}/.ko.yaml <<EOF
-builds:
-- id: cloud-controller-manager
-  dir: .
-  main: cmd/cloud-controller-manager
-  ldflags:
-  - -X k8s.io/component-base/version/verflag.programName=cloud-controller-manager
-
-  - -X k8s.io/component-base/version.buildDate=${BUILD_DATE}
-  - -X k8s.io/component-base/version.gitCommit=${KUBE_GIT_COMMIT}
-  - -X k8s.io/component-base/version.gitMajor=${KUBE_GIT_MAJOR}
-  - -X k8s.io/component-base/version.gitMinor=${KUBE_GIT_MINOR}
-  - -X k8s.io/component-base/version.gitTreeState=${KUBE_GIT_TREE_STATE}
-  - -X k8s.io/component-base/version.gitVersion=${KUBE_GIT_VERSION}
-
-  - -X k8s.io/client-go/pkg/version.buildDate=${BUILD_DATE}
-  - -X k8s.io/client-go/pkg/version.gitCommit=${KUBE_GIT_COMMIT}
-  - -X k8s.io/client-go/pkg/version.gitMajor=${KUBE_GIT_MAJOR}
-  - -X k8s.io/client-go/pkg/version.gitMinor=${KUBE_GIT_MINOR}
-  - -X k8s.io/client-go/pkg/version.gitTreeState=${KUBE_GIT_TREE_STATE}
-  - -X k8s.io/client-go/pkg/version.gitVersion=${KUBE_GIT_VERSION}
-EOF
-
-# KO_CONFIG_PATH is the directory, not the path, until https://github.com/ko-build/ko/pull/731 lands in a tag (it's not in 0.12)
-KO_CONFIG_PATH=${WORKDIR}/ go run github.com/google/ko@v0.12.0 build --tags ${IMAGE_TAG} --base-import-paths --push=true ./cmd/cloud-controller-manager/
+echo "git status:"
+git status
+
+echo "Configuring docker auth with gcloud"
+gcloud auth configure-docker
```

Comment:
ah I had the same issue when I run docker build image locally: https://github.com/kubernetes/cloud-provider-gcp/pull/825#issuecomment-2761981878, ko was fine. 

But the push image CI does not have permission issue to run docker build image, so I suspect this may or may not be necessary. But does not hurt to add.